torch/csrc/jit/python/python_ir.cpp

```diff
@@ -526,6 +526,7 @@ void initPythonIRBindings(PyObject* module_) {
           "Find all nodes",
           py::arg("kind"),
           py::arg("recurse") = true)
+      .def("updateLoopInputs", [](Node& n) { return updateLoopNodeInputs(&n); })
```

Comment:
I'm not sure how much this is improving the design, then simply rename `_jit_pass_fixup_onnx_loop_node_inputs` in `python/init.cpp` and drop the word `pass`.. 
To refactor it completely, I think you may need to consider re-implement this part in python, and have loop conversion code in utils.py call it. So that this entire part is moved out of c++.